content/blog/wingingit/wingingit.11tydata.yaml

```
@@ -0,0 +1,6 @@
+venue: The Winging It Channel
```

I think we could keep this short and simple:
```suggestion
venue: Winging It
```

---

Ok. To me this seems like a broader concern about the language in the byline. This is not a new issue specific to our podcast, but could be applied to [most of the podcasts we've posted](https://www.oddbird.net/tags/podcast/). I'd prefer a solution that we like across all podcasts, without needing to change their names or add "The `<x>` Channel" or "The `<x>` Podcast" to each title.

---

@stacyk @mirisuzanne What do you think of the changes in https://github.com/oddbird/oddleventy/commit/09980562e9ee0dfae022d5464de9f84bd7e54405?

---

works for me 👍🏼

---

Some turn out a bit strange... https://deploy-preview-378--oddleventy.netlify.app/2022/04/14/linkedin/ is his name  the name of the show?

---

Like this one might make more sense to say "on Design Systems Advocate" instead of "on Jina": 
https://deploy-preview-378--oddleventy.netlify.app/2020/05/05/jina-ama/

---

@stacyk I noticed that too, but I don't think it's any stranger than the existing version? https://www.oddbird.net/2022/04/14/linkedin/

---

I suppose that is true